apps/web/src/app/[locale]/challenge/[slug]/solutions/_components/solution-detail.tsx

```diff
@@ -53,11 +58,11 @@ export function SolutionDetails({ solution }: Props) {
         <div className="custom-scrollable-element flex-1 overflow-y-auto px-4 pb-16 pt-3">
           <div className="mb-5 flex flex-col gap-3">
             <div className="flex items-center gap-3">
-              <Link href={`/challenge/${slug}/solutions`} className="h-7 w-7">
+              <button onClick={handleClickLeftIcon} className="h-7 w-7">
```

Comment:
you're able to visit solutions directly. this wouldnt work?

Reply:
When I implement the code, I added the history back behavior because it's a back icon.
Now that I think about it, link component would be a better way to do it
We'll revert the code back.